Extension/package.json

```diff
@@ -234,7 +235,7 @@
           "description": "%c_cpp.configuration.autoAddFileAssociations.description%",
           "scope": "resource"
         },
-        "C_Cpp.workspaceParsingPriority": {
+        "C_Cpp.workspaceIndexing.priority": {
```

Comment:
This really does apply to the "parsing" and not the "indexing". Was there was other reason for this change?